elements/pl-file-upload/pl-file-upload.js

```diff
@@ -168,6 +168,8 @@ window.PLFileUpload.prototype.renderFileList = function() {
             $fileStatusContainerLeft.append('<p class="file-status">uploaded</p>');
         }
         if (fileData) {
+            var download = '<a download="' + fileName + '" class="btn btn-outline-secondary btn-sm" id="file-download-' + uuid + '-' + index + '" onclick="event.stopPropagation();" href="data:application/octet-stream;base64,' + fileData + '">Download</a>';
```

Comment:
Is the `id` attribute necessary?

Separately, this should be all you need for proper spacing:

```suggestion
            var download = '<a download="' + fileName + '" class="btn btn-outline-secondary btn-sm mr-2" id="file-download-' + uuid + '-' + index + '" onclick="event.stopPropagation();" href="data:application/octet-stream;base64,' + fileData + '">Download</a>';
```

Reply:
I'd suggest `mr-1`, for consistency with the spacing used in `pl-file-preview`.

Reply:
@jonatanschroeder is the onclick necessary here?  (I copied your code, but wasn't sure.)

Reply:
The `onclick` is to keep the preview from opening. It is to avoid the button click to trigger the opening of the preview, which is caused by clicking anywhere on the container of this link, including the download button.

Reply:
(answered my own question about the onclick...)

Reply:
da39769 adds mr-1 and removes the id.